src/sentry/event_manager.py

```diff
@@ -599,6 +612,18 @@ def save(
                     project=project, event=job["event"], sender=Project
                 )
 
+            if (
+                has_event_minified_stack_trace(job["event"])
+                and not project.flags.has_minified_stack_trace
+                # We only want to record events from projects created after 2022-12-14,
+                # otherwise amplitude would receive a large amount of data in a short period of time
+                and project.date_added
+                > START_DATE_TRACKING_FIRST_EVENT_WITH_MINIFIED_STACK_TRACE_PER_PROJ
```

Comment:
@priscilawebdev So we should really be setting the `has_minified_stack_trace` on the project regardless of the project creation time. What we should be skipping is sending the analytics events for those projects while still updating the flags. Can someone on the telemetry team make this change cc @maciej-ka